source/configuration_manual/shared_mailboxes/cluster_setup_example.rst

```diff
@@ -0,0 +1,97 @@
+.. _mailbox_sharing_in_cluster_simple_example:
+
+===============================================================
+Example config snippets for shared mailboxes in Dovecot Cluster
+===============================================================
+
+The following snippets show important configuration parts for configuring
+cluster wide sharing of mailboxes. The following snippets are not complete
+configuration. For more details please refer to
+:ref:`mailbox_sharing_in_cluster`.
+
+
+Dovecot Proxy configuration snippet
+-----------------------------------
+
+.. code-block::
+
+        passdb {
+          args = proxy=y password=masterpass host=<director-ip>
+          driver = static
+        }
+
+Dovecot Director configuration snippet
+--------------------------------------
+
+.. code-block::
+
+        director_servers = <director-ip>
+        director_mail_servers = <backend-ip>
+
+        passdb {
+          driver = static
+          master = yes
+          default_fields = proxy=y
+          args = password=imapcpass
+        }
+
+        passdb {
+          driver = static
+          args = proxy=y password=masterpass
+        }
+
+Dovecot Backend configuration snippet
+--------------------------------------
+
+.. code-block::
+
+        mail_plugins = $mail_plugins acl
+
+        imapc_host = <director-ip>
+        imapc_password = imapcpass
+        imapc_user = # empty defaults to shared user
+        imapc_features = fetch-bodystructure fetch-headers rfc822.size search modseq acl
+
+        namespace shared {
+          type = shared
+          prefix = shared/%%u/
+          separator = /
+          list = children
+          location = imapc:~/shared/%%u:INDEXPVT=~/shared-pvt/%%u
```

Comment:
From a quick discussion with @cmouse on IRC today, I understand that to avoid having the subscription messed up by the user accessing the shared mailbox, one need to have `subscriptions = no` on the shared namespace?
If so, maybe it should be added here (and on the original blob in the non-changed part above)?

Reply:
Yes right thank you, it should be mentioned here like it says here: https://doc.dovecot.org/configuration_manual/namespace/#namespace-subscriptions